src/semgrep_agent/semgrep_app.py

```diff
@@ -153,14 +153,19 @@ def report_failure(self, error: SemgrepError) -> int:
     def report_results(self, results: Results) -> None:
         debug_echo(f"=== reporting results to semgrep app at {self.url}")
 
+        fields_to_omit = constants.PRIVACY_SENSITIVE_FIELDS.copy()
+
+        if "pr-comment-autofix" in os.getenv("SEMGREP_AGENT_OPT_IN_FEATURES", ""):
+            fields_to_omit.remove("fixed_lines")
+
         response: Optional["requests.Response"] = None
 
         response = self.session.post(
             f"{self.url}/api/agent/scan/{self.scan.id}/findings",
             json={
                 "token": os.getenv("GITHUB_TOKEN"),
```

Comment:
Also perhaps add a comment here that this is now just a backup method of authentication, only used if github.get_install_token fails

Reply:
```suggestion
                "token": os.getenv("GITHUB_TOKEN"),  # backup token in case the app is not available
```